cmd/antrea-agent/bincover_run_main_test.go

```diff
@@ -0,0 +1,13 @@
+// +build testbincover
+
+package main
+
+import (
+	"testing"
+
+	"github.com/confluentinc/bincover"
```

Comment:
good job finding this package, I hope it's actively maintained

Reply:
I'm surprised by the large number of dependencies in their go.sum file, which in turn is causing a large number of new entries in our go.sum file.

Out-of-curiosity, I tried running `go mod tidy` after cloning the `github.com/confluentinc/bincover` repo, and a *very* large number of entries went away. I can try submitting a PR there to clean things up.

Reply:
https://github.com/confluentinc/bincover/pull/15

Reply:
ok, let me also try again.

Reply:
thanks Antonin.

Reply:
I don't thin this is resolved? We are still adding a ton of checksums to our go.sum that we are not actually using because bincover does not actually use them. The maintainer of bincover just commented today on my PR (https://github.com/confluentinc/bincover/pull/15) that he would review it tomorrow. Hopefully we can get it merged soon and avoid polluting our own go.sum file like this.

Reply:
@lxiaopei my PR was accepted and merged, please update this PR to use the latest version of bincover